custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/README.md

```diff
@@ -41,13 +40,32 @@ env:
       fieldRef:
         fieldPath: metadata.namespace
 ```
+
+### PodMonitoring Resource
```

Comment:
I'm not sure if this example belongs here. This file is explaining how to use prom-to-sd but your section is related to GMP.
If we want to add a section like this it should be more explicit that this is an alternative to prom-to-sd entirely and properly link to the GMP docs for setting everything up.

Reply:
oh I see, yea i think you are right, I will do it that way